lib/galaxy/util/commands.py

```diff
@@ -110,9 +111,10 @@ def argv_to_str(command_argv, quote=True):
     return " ".join(map_func(c) for c in command_argv if c is not None)
 
 
-def _wait(cmds, **popen_kwds):
+def _wait(cmds, input=None, **popen_kwds):
     p = subprocess.Popen(cmds, **popen_kwds)
-    stdout, stderr = p.communicate()
+    stdout, stderr = p.communicate(input)
+    stdout, stderr = unicodify(stdout).strip(), unicodify(stderr).strip()
```

Comment:
I'm not sure we should `strip()` here, maybe sometime spaces are significant?

Reply:
Done and strip restored at original places.